network/transparent_endpointclient_linux.go

```diff
@@ -246,11 +252,13 @@ func (client *TransparentEndpointClient) ConfigureContainerInterfacesAndRoutes(e
 	// arp -s 169.254.1.1 e3:45:f4:ac:34:12 - add static arp entry for virtualgwip to hostveth interface mac
 	log.Printf("[net] Adding static arp for IP address %v and MAC %v in Container namespace",
 		virtualGwNet.String(), client.hostVethMac)
-	if err := client.netlink.AddOrRemoveStaticArp(netlink.ADD,
-		client.containerVethName,
-		virtualGwNet.IP,
-		client.hostVethMac,
-		false); err != nil {
+	linkInfo := netlink.LinkInfo{
+		Name:       client.containerVethName,
+		IPAddr:     virtualGwNet.IP,
+		MacAddress: client.hostVethMac,
+	}
+
+	if err := client.netlink.SetOrRemoveLinkAddress(linkInfo, netlink.ADD, netlink.NUD_PERMANENT); err != nil {
```

Comment:
i meant have to change only for transparent singletenancy and not in other places. Lets update to NUD_PROBE here but not in other places

Reply:
Okay my bad, just updated it.